library/pdns_zone.py

```diff
@@ -191,7 +197,9 @@ def zone_exists(module, base_url, zone):
         return None
 
     if info['status'] != 200:
-        module.fail_json(msg="failed to check zone %s at %s: %s" % (zone, url, info['msg']))
```

Comment:
why not single line? Here I find it is clearer ...
(there are a few of these further)

Reply:
The issue is line length.